assets/stylesheets/utilities/_responsive.scss

```diff
@@ -6,6 +6,10 @@
     width: 100%;
 }
 
+.responsive-image {
```

Comment:
The `u-responsive` class on an img does the same thing below. If that's tricky to remember, let's add it in the setup below so we still have the `u` prefix and the size namespaces.